src/evict/evict_page.c

```diff
@@ -675,63 +677,71 @@ __evict_review(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t evict_flags, bool
             LF_SET(WT_REC_SCRUB);
     }
 
-    /* Acquire a snapshot if coming through eviction thread route. */
+    /*
+     * Acquire a snapshot if coming through eviction thread route. Also, if we have entered eviction
```

Comment:
Text fix: ...if coming through the eviction thread...

Reply:
Fixed in 65be4b5.